bayes_opt/acquisition.py

```diff
@@ -299,17 +309,21 @@ class UpperConfidenceBound(AcquisitionFunction):
 
     """
 
-    def __init__(self, kappa=2.576, exploration_decay=None, exploration_decay_delay=None, random_state=None) -> None:
-        super().__init__(random_state=random_state)
-        assert kappa >= 0, "kappa must be greater than or equal to 0."
+    def __init__(
+        self, kappa=2.576, exploration_decay=None, exploration_decay_delay=None, random_state=None
+    ) -> None:
+        if kappa < 0:
+            error_msg = "kappa must be greater than or equal to 0."
+            raise ValueError(error_msg)
```

Comment:
It's good that you caught this case -- could you please also add a test case to cover it?

Reply:
This is the code i added by removing `assert` from the existing code.
I'll add the test code soon.